src/compiler/scanner.ts

```diff
@@ -62,77 +62,77 @@ namespace ts {
     }
 
     const textToToken = createMapFromTemplate({
-        "abstract": SyntaxKind.AbstractKeyword,

```

Comment:
this seems too much.. not sure if we even need this rule.

Reply:
Consistency has some value here. some of these items will have to be quoted, so let them all be quoted..

Reply:
Changed to the [consistent-as-needed](https://palantir.github.io/tslint/rules/object-literal-key-quotes/) option.